docs/getting-started.md

```diff
@@ -1,12 +1,12 @@
 # Scully - Getting Started
 
 ## Prerequisites 
-The first thing you need to get started with Scully is a working Angular app using **Angular 9.x.x**.
+The first thing you need to get started with Scully is a working Angular app using **Angular 9.x.x** and **Node 12.x.x**  
 
 You can create a new Angular 9 app using the following command:
 
 ```bash
-npx -p @angular/cli@next ng new my-scully-app
+npm install -g @angular/cli@next && ng new my-scully-app
```

Comment:
What kinds of problems did you have with npx?

Reply:
I prefer using npx so you don't have to change your global @angular/cli version

Reply:
I agree, we should not recommend installing the `next` version globally. One of the great features of npx is that it allows doing just this, without having the need of updating global deps.

Reply:
Well that’s why we do use of nvm, right? I didnt play well with npx in here. I would consider installing angular next using nvm in a node 12.x environment (since it’s required by angular 9)

Reply:
I think docs like these should advise the most common use-case. 

More advanced devlopers are able to analyse the instructions and adjust to them to their use case (while understanding the risks).

Reply:
Let's add both cases.... 

using: 
`npx -p @angular/cli@next ng new my-scully-app`  or  `npm install -g @angular/cli@next && ng new my-scully-app` . 

@beeman  what do you think?

Reply:
@Villanuevand I stay with the point that we should not recommend installing it globally. If we do we should mention it might lead to potential compatibility issues.

Now, the Angular CLI is smart enough to use the local version of the project. So if you have v9 installed globally, it will run the version local to the project you're on (like v8 or v7), but there might be unintentional effects when running `ng new` another time.  

I got called back by someone from the Angular team a while back when I blogged about installing `next` globally`, but it seems to happen all the time.

Reply:
npx here wont work.  try it following the docs.

Reply:
How about something like this:

---

```
npx -p @angular/cli@next ng new my-scully-app
```

If that fails, you can install the `next` version of Angular CLI globally and create a new app with that version. 

**Note**: doing this means that any new apps you will create after this will use version 9.

```
npm install -g @angular/cli@next
ng new my-scully-app
```

---

Reply:
makes sense